### PR TITLE
[15.0][MIG] sequence_check_digit: Migration to 15.0

### PR DESCRIPTION
**Pull request by kaynnan**
cc @marcelsavegnago 

#390

### CODE REVIEW

**Comment by marcelsavegnago** [comment]
@augustodinizl @aug1804 Can you do the functional review please?

File: sequence_check_digit/models/ir_sequence.py [hunk 1]
```diff
@@ -37,17 +37,17 @@ def check_check_digit_formula(self):
         try:
             self.get_next_char(0)
         except Exception:
-            raise ValidationError(_("Format is not accepted"))
+            raise ValidationError(_("Format is not accepted")) from None
```
Comment (etobella) [inline sequence_check_digit/models/ir_sequence.py:40]: Why did you add the from None?

Comment (kaynnan) [reply sequence_check_digit/models/ir_sequence.py:40]: Due to the suggestion made by the pre-commit
